src/Stratis.Features.FederatedPeg/Interfaces/IFederationWalletManager.cs

```diff
@@ -60,13 +60,6 @@ public interface IFederationWalletManager
         /// <returns>A value indicating whether this transaction affects the wallet.</returns>
         bool ProcessTransaction(Transaction transaction, int? blockHeight = null, Block block = null, bool isPropagated = true);
 
-        /// <summary>
-        /// Removes a transaction not yet broadcasted or included in a block.
-        /// </summary>
-        /// <param name="transaction">The transaction to remove.</param>
-        /// <returns>A value indicating whether this transaction affects the wallet.</returns>
-        bool RemoveTransaction(Transaction transaction);
```

Comment:
This method didn't have a lock in it, just want to be sure it's not getting called from any addons or anything